docs/src/kubernetes_upgrade.md

```diff
@@ -1,54 +1,109 @@
-to clean up the technical debt and reduce the business risks, despite
-the introduction in your Kubernetes infrastructure of controlled
-downtimes that temporarily take out a node from the cluster for
-maintenance reasons (recommended reading:
-["Embracing Risk"](https://landing.google.com/sre/sre-book/chapters/embracing-risk/)
-from the Site Reliability Engineering book).
+Updating Kubernetes involves planning and executing maintenance tasks, such as
+applying security updates to underlying Linux servers, replacing malfunctioning
+hardware components, or upgrading the cluster to the latest Kubernetes version.
+These activities are essential for maintaining a robust and secure
+infrastructure.
 
-For example, you might need to apply security updates on the Linux
-servers where Kubernetes is installed, or to replace a malfunctioning
-hardware component such as RAM, CPU, or RAID controller, or even upgrade
-the cluster to the latest version of Kubernetes.
+## Maintenance Operations in a Cluster
 
-Usually, maintenance operations in a cluster are performed one node
-at a time by:
+Typically, maintenance operations are carried out on one node at a time, following a [structured process](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/):
 
-1. evicting the workloads from the node to be updated (`drain`)
-2. performing the actual operation (for example, system update)
-3. re-joining the node to the cluster (`uncordon`)
+1. eviction of workloads (`drain`): workloads are gracefully moved away from
+   the node to be updated, ensuring a smooth transition.
+2. performing the operation: the actual maintenance operation, such as a
+   system update or hardware replacement, is executed.
+3. rejoining the node to the cluster (`uncordon`): the updated node is
+   reintegrated into the cluster, ready to resume its responsibilities.
 
-The above process requires workloads to be either stopped for the
-entire duration of the upgrade or migrated to another node.
+This process requires either stopping workloads for the entire upgrade duration
+or migrating them to other nodes in the cluster.
 
-While the latest case is the expected one in terms of service
-reliability and self-healing capabilities of Kubernetes, there can
-be situations where it is advised to operate with a temporarily
-degraded cluster and wait for the upgraded node to be up again.
+## Temporary PostgreSQL Cluster Degradation
 
-In particular, if your PostgreSQL cluster relies on **node-local storage**
-\- that is *storage which is local to the Kubernetes worker node where
-the PostgreSQL database is running*.
-Node-local storage (or simply *local storage*) is used to enhance performance.
+While the standard approach ensures service reliability and leverages
+Kubernetes' self-healing capabilities, there are scenarios where operating with
+a temporarily degraded cluster may be advisable. This is particularly relevant
```

Comment:
"... may be advisable" ... doesn't sound right.
Perhaps I'm misunderstanding. @gbartolini perhaps you meant "acceptable" not "advisable"?